.gitignore

```diff
@@ -7,6 +7,8 @@ upload-files/
 demo*
 gen-java
 *.class
+swagger-ui-*
+hugegraph-dist/dist.sh
```

Comment:
Thanks, but we need fix it in the dist-script rather than exclude them in git (will be fixed it in another PR) @VGalaxies 

![yzBIrgUZmN](https://github.com/apache/incubator-hugegraph/assets/17706099/6a85e38c-bffd-4289-a555-33f521f588b2)

Reply:
ok, fixed and rollback change